src/background.js

```diff
@@ -1,24 +1,20 @@
-let namespace = window.browser || chrome
+const runtime = chrome || window.browser
+
+//register handler for communication with content script
+runtime.runtime.onMessage.addListener((request, sender, sendResponse) => {
```

Comment:
`runtime.runtime` sounds weird. Maybe we should call it `browser`.

Reply:
browser is protected. `runtime` is used in many docs and examples, that is why I used it.